### ISSUE TITLE
[Bug] The detail value is below the gauge

### ISSUE THREAD

**Issue by cavanierc** [apache/echarts#19971]
### Version

5.5.0

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?c=gauge-temperature

### Steps to Reproduce

On echart documentation, on the [gauge](https://echarts.apache.org/examples/en/editor.html?c=gauge-temperature), if you shrink the browser window enough, the value of the gauge detail falls below the gauge itself.

### Current Behavior

<img width="275" alt="Capture d’écran 2024-05-23 à 15 37 36" src="https://github.com/apache/echarts/assets/34306760/f552d26d-adfe-468a-85c1-0827cb09746f">

The value is below the gauge

### Expected Behavior

We need to be able to see the value of the gauge above the gauge, so that the value is legible.

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

### Any additional comments?

The problem seems to be that the svg components are displayed in the order in which they were created, so the text would have to be generated last for it to be above the gauge.

**Comment by helgasoft** [comment]
The referenced example is made of **two gauges**. By setting _zlevel: 11_ to the second one, we can improve the view, but not solve the problem entirely, unless the second _progress_ is removed, [Demo](https://echarts.apache.org/examples/en/editor.html?c=gauge-temperature&code=PYBwLglsB2AEC8sDeAoWsDOBTAThLGAXLANprrLkWxgCeIWxA5AOYCGAri1kwDRUUAxlmhhcxEkwCsABgCkfWEwBs8pgF1-1dBjBscYAILQWAG0awATDJlbtIgCbGzFgLTW71ALYRoxWwLoXmwAHsSqnhQYIKYQYAByHF4ARuKwAIyWkehxWF4AynTmxKjaQsCmwDjMAMQAYnWGAEIAnOlMgbAAvtmwIDjALDgERMiYABbAAO7EYDgcWLywUxAOYOPEAMwy3b0gwL5i1ZRlOpMzsABmbKbYnT2doRAYADK-FqWnsdBYhbTFJ1O6BWaw2sG2nXQXXuvSeGAAKhBBABrEqQ2AOZ56aDCYiuAAsUl6OhicUSKTSRPR31-RQ-6OBq3WxCyDNgggqVVqLR5HSB0LKDzK0ViYDePzRQMxujYOLcUlZQPMJmZGXxxNgNL-AM-QJBqs2GvQHMqxyYNR5LT5pwF2iF2jhLzYqVMktO0uxuNg7gCQJNXKUFt5GsuMDA-QgAC8LNYYY8cZNjrrtBhzsRrrcsHGypAwDr0anpumbndBb0HFg9BBXWNCxcM9hdp0HGw9BIkLAAG43BYsmQAOnx-O66gE9vQyfQdAYzHYXB4vUj5k7WBr6XSvWEojSklkCiWKjUmk6MoMzgB1l99mgThMAJ9vR8flgV-8oXCr4ouQKdLdZX9Zr1AAIgA7Js-Igda1DjhQ_SDMMGCjB2wArjgphsCAxaZksdazPMizLEyYIABxNmU-yHGkk5RGmVwllmZaPCEzzivSQK4XRmbZg6zEIkiqKAsKtENgxdq9CKcSsX-KbCfR3HUI6zqrtJ1AcSJ8kUBWVY1tR6DdqYCzGBAwSQDAeELBq-pgoe-7Uu8AASWAQCw4xgMQ-KftQyRVBWOAAEpsJiHCjMRGrAJclzYGAADCIhHBIthKK46RSAox5AqGogRtGH4hmGADqTkuW5SjeaYvl8OioY4CZ8VKEg-kLF0sAAA3RZVfqcmawwOEwsAAPT9UwvjjLgcRQRQMHoC2bakB2jUxgOQ4jmO5Cjl0ADcA0AFQoFFACS244PpAAUlwcDiplwCdACUgIctAuiwDgsoOMAXgILAADUJ0ALKtuM_YvTe723bA22wKoN39mAwB1BAIRYA4J2WDdG3kF4tDReM-hgP2UUAPLgFA0Ancm2B4AQEidLpGKtmw1NAhObJ6T2FjA29XhsraZSjoxZS0zNDOkGytPUAtxAc-93PonzdoCHLXRoygPRWDYMho9t_VAA).
![gauge-temperature (1)](https://github.com/apache/echarts/assets/13038071/f7e40454-fe41-4ec6-a973-383e19128ed6)

**Comment by cavanierc** [comment]
Yes, it allows you to pass in front of the background gauge, but my problem is that it would have to pass in front of the progress gauge. Is it possible to set a specific zlevel on the progress gauge?

**Comment by helgasoft** [comment]
ok, if **two** do not work, let it make **three** - [Demo](https://echarts.apache.org/examples/en/editor.html?c=line-simple&code=M4MwBAvGDebAFgewO4C4wgIYBtgFMwBfAbgCgATANxygCYAGAOgBZmyBzd2yGUsMAC4BPAA550AcnaYAruzwSANHzABjPADsBeAE7oA2hICs9AKRKwEgGxmJAXWX9gAzDoEBBDe2ziwDeo5gmuSe3r4AtAyBALaYAB7oNoEiiACWWrrooIHxqcAAMum-2Sq5wAAqqaoA1lkggcAi2KkChRrF9aVxefmYAEZ42HXKKgBePpSD6ACM04EteNEAysI-6LCqiNiIepYAxABiACIA7ADMzCcSRMk6iOw6eMDA62CIkzrYmCLoWLh4ijgSDQgh0MgBYGQqXIAng6AAHDcVOQ8C5UkMYECUL8cPgkfxyJgXAZYNRsOD0FQcEQ7KQSKROGceHhVPBXAJgIwZAJ0YxVNt2gAKTi0ACUHHYZ0YIjuDyewB4oAlUpRaOwPGgKjJ4M8qViPMQGnQAjBAJUUJhcMsNnMgWa7QAEnhUux4AJ0MwAio-jsUToAEqYcipGQvMDwwKIEAgfACADCmm0u30AUs4WmRnMDhUIENAiWqVGvmmURzeYA6s7Xe7LD7sH6lGWdPqk5JoNq8IQwAAGuON_ibba7CSPcgSOlkUiIEQGjQa0oaPVE1KGnH_Bq6VJPAwqfia_gHwSiXxSWTyfuHtSJzJgQwmW3W2zZy9wFxuUJrPz0L0v4IfiKli-sQJGASS7gejTNAIAByMjRAMuwloEh4LMsqy-BsWw7JIhwHO4ABCACc0zXIQyEHjK9yPM8rwINioLgoCFqwugZz0Pil4pOkrZwJ0L6YBorLYbx5H8DyAifiU4H8GUbQYWA9p4CsQifrAzFWmxRAcYeZSVDU6zSQewbOAJ6joOEzBGKJEFNC0sHwTeVmGfwinKapkLQixfiAoOwkSHshGBaRhlkYZkEtHJBkvgSeQuIJERGLQ1kuZo7BedMzDJQpRRufJ6msT5WHDgFQVECF1myf0Uy8NFYDGXFZlgJEP7Rb5xWBYRF4vrmWgFkW6AMOVhmqpg6K0cCa54qFL6EsSt6kjgFJ1WSNLgdNB4iuRjJgCotIkEAA).
lot of trouble for a pretty picture, heh...
![line-simple](https://github.com/apache/echarts/assets/13038071/e446d79a-10ba-4db8-815b-e3cea89bd304)
